pom.xml

```diff
@@ -16,34 +16,40 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>11</java.version>
+		<java.version>21</java.version>
 
 		<main.class>uk.gov.companieshouse.insolvency.delta.InsolvencyDeltaConsumerApplication</main.class>
 
-		<map-struct.version>1.4.2.Final</map-struct.version>
+		<map-struct.version>1.6.3</map-struct.version>
 		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-		<maven-build-helper-plugin.version>3.2.0</maven-build-helper-plugin.version>
-		<maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
-		<docker-maven-plugin.version>1.0.0</docker-maven-plugin.version>
-		<ch-kafka.version>1.4.2</ch-kafka.version>
-		<structured-logging.version>1.9.37</structured-logging.version>
-		<kafka-models.version>1.0.28</kafka-models.version>
-		<private-api-sdk-java.version>2.0.454</private-api-sdk-java.version>
-		<api-sdk-manager-java-library.version>1.0.4</api-sdk-manager-java-library.version>
-		<jib-maven-plugin.version>3.1.1</jib-maven-plugin.version>
-		<io-cucumber.version>7.2.3</io-cucumber.version>
+		<maven-build-helper-plugin.version>3.5.2</maven-build-helper-plugin.version>
+		<maven-failsafe-plugin.version>3.5.2</maven-failsafe-plugin.version>
+		<docker-maven-plugin.version>1.6.0</docker-maven-plugin.version>
+		<ch-kafka.version>3.0.3</ch-kafka.version>
+		<structured-logging.version>3.0.20</structured-logging.version>
+		<kafka-models.version>3.0.8</kafka-models.version>
+		<private-api-sdk-java.version>4.0.250</private-api-sdk-java.version>
+		<api-sdk-manager-java-library.version>3.0.6</api-sdk-manager-java-library.version>
+		<jib-maven-plugin.version>3.4.4</jib-maven-plugin.version>
+		<io-cucumber.version>7.20.1</io-cucumber.version>
 		<springfox-boot-starter.version>3.0.0</springfox-boot-starter.version>
```

Comment:
Springfox doesn't support Spring Boot 3.x, you'll need to migrate this
https://springdoc.org/migrating-from-springfox.html